### PR TITLE
Added support for project_value_buffer AND dealloc_value_buffer instruction

### PR DESCRIPTION
**Pull request by mmroz**

### CODE REVIEW

File: lib/WALASupport/SILWalaInstructionVisitor.cpp [hunk 1]
```diff
@@ -389,6 +389,35 @@ jobject SILWalaInstructionVisitor::visitEndUnpairedAccessInst(EndUnpairedAccessI
...
+jobject SILWalaInstructionVisitor::visitProjectValueBufferInst(ProjectValueBufferInst *PVBI) {
+  SILValue BufferValue = PVBI->getOperand();
+  string ValueTypeName = PVBI->getValueType().getAsString();
+
+  if (Print) {
+    llvm::outs() << "\t [BUFFER ADDR]: " << BufferValue.getOpaqueValue() << "\n";
+    llvm::outs() << "\t [VALUE TYPE]: " << ValueTypeName << "\n";
+  }
+
+  if (SymbolTable.has(BufferValue.getOpaqueValue())) {
```
Comment (karimhamdanali) [inline lib/WALASupport/SILWalaInstructionVisitor.cpp]: What if the `Buffervalue` is not in the symbol table? Shouldn't we insert it there?

Comment (mmroz) [reply lib/WALASupport/SILWalaInstructionVisitor.cpp]: I originally thought the same but all of the ‘project’ instructions state that they are undefined behaviour if they do not exist so I don’t think so!

Comment (karimhamdanali) [reply lib/WALASupport/SILWalaInstructionVisitor.cpp]: I see. Could you please add that comment in the code as a note or something, for future reference? I'll approve after that change is added to the PR.

**Comment by mmroz** [comment]
I added comments to each 'Project' instruction

**Review by karimhamdanali — APPROVED** [review]
LGTM

**Pull request merged**